index.js

```diff
@@ -250,3 +266,15 @@ function initSession (opts, fn) {
     fn(uid, new Session(opts));
   });
 }
+
+app.on('open-file', (event, path) => {
+  let lastWindow = app.getLastFocusedWindow();
```

Comment:
This can be a `const`. `getLastFocusedWindow` doesn't need to necessary live in `app`